packages/Organisms/CampaignSecondary/index.vue

```diff
@@ -116,6 +120,10 @@ export default {
         padding: $rpl-campaign-secondary-content-padding-s;
       }
     }
+    &__video {
+      .rpl-embedded-video-iframe-container iframe {border: 0}
```

Comment:
Hey @dylankelly this reminds me the sass lint is not working in vue template file. If it works, it will throw warning I think. Suggested way is:
```
.rpl-embedded-video-iframe-container {
  iframe {
    border: 0;
  }
}
```
Another thing I think is this css may should be added to `RplEmbeddedVideo` component which shouldn't have a border, right?

Reply:
ah, from your solution direction it sounded like you just wanted this on the campaign secondary.  Have updated.